src/overlay005/fieldmap.c

```diff
@@ -426,10 +426,10 @@ static BOOL FieldMap_ChangeZone(FieldSystem *fieldSystem)
     }
 
     {
```

Comment:
**polish:** We're also working on cleaning up unnecessary scope blocks like these.

Reply:
Looks like there's still a couple pesky extra scope blocks in this function.

Reply:
Should be taken care of now